app/controllers/api/v1/validation_requests_controller.rb

```diff
@@ -13,4 +13,10 @@ def check_token_and_set_application
       @planning_application
     end
   end
+
+  def check_file_size
+    if params[:new_file].size > 30.megabytes
+      render json: { "message": "The file must be 30MB or less" }, status: :bad_request
+    end
+  end
```

Comment:
we should add a little test for this I guess - did you ever find out how to stub the size on a spec?

Reply:
Steph recommended leaving the test for now as it was blocking getting this in

Reply:
... I may move this to a WIP PR while we think about it

Reply:
Couldn't crack a test :) Somewhere in the upload/request chain, ActionDispatch re-reads the file and erases my `size` mock (though it appears in the `Content-Length` header, but we want to check `size`, it's easier).